multi-int-store/build.gradle

```diff
@@ -4,6 +4,7 @@ plugins {
 }
 
 dependencies {
+	implementation "org.springframework.data:spring-data-solr:4.0.8.RELEASE"
```

Comment:
Out of curiosity why are we using this version of the spring framework?

Reply:
Newest version